src/main/kotlin/com/demonwav/mcdev/platform/mcp/color/inspections/ColorComponentOutOfRangeInspection.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2018 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.mcp.color.inspections
+
+import com.demonwav.mcdev.facet.MinecraftFacet
+import com.demonwav.mcdev.platform.mcp.McpModuleType
+import com.demonwav.mcdev.platform.mcp.color.McpColorMethod
+import com.demonwav.mcdev.platform.mcp.color.McpColorMethods
+import com.demonwav.mcdev.platform.mcp.color.McpColorResult
+import com.demonwav.mcdev.platform.mcp.color.McpColorWarning
+import com.demonwav.mcdev.platform.mcp.color.findColors
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.module.ModuleUtilCore
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiCallExpression
+import com.siyeh.ig.BaseInspection
+import com.siyeh.ig.BaseInspectionVisitor
+import com.siyeh.ig.InspectionGadgetsFix
+import org.jetbrains.annotations.Nls
+
+class ColorComponentOutOfRangeInspection : BaseInspection() {
```

Comment:
Can we avoid introducing new inspections built on `BaseInspection`? In my opinion that class is pretty much obsolete and most things can be implemented with one of IntelliJ's base classes in a nicer way.